code/modules/surgery/organs/vocal_cords.dm

```diff
@@ -285,8 +285,9 @@
 	//SILENCE
 	else if((findtext(message, silence_words)))
 		cooldown = COOLDOWN_STUN
+		var/max_silent_time = 1 MINUTES
```

Comment:
this should be a define directive
```
#define VOICE_OF_GOD_MAX_SILENCE_TIME 1 MINUTES
```
at the top of the file
then
```
#undef VOICE_OF_GOD_MAX_SILENCE_TIME
```
at the bottom